plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -425,10 +480,12 @@ class CodeWhispererCodeScanManager(val project: Project) {
      *   2. Update the lookups - [fileNodeLookup] for efficiently adding scan nodes and
      *   [scanNodesLookup] for receiving the editor events and updating the corresponding scan nodes.
      */
```

Comment:
nit: move this comment below `codeScanTreeNodeRoot` so that it is directly above `createCodeScanIssuesTree`